packages/@misk/simpleredux/src/utilities/index.ts

```diff
@@ -139,7 +139,7 @@ export const getFirstTag = <
   [key: string]: UNIONED_TYPE
 }): T => {
   if (Object.keys(payload).length === 1) {
-    return (payload[Object.keys(payload)[0]] as unknown) as T
```

Comment:
sorta wish we could keep these clarifying parens

Reply:
hmm I can see if there's a rule to keep these around

Reply:
There's not really a way to configure this specifically (prettier is very opinionated regarding it's style). However, I've rewritten this line in a way that I think is clearer to begin with!